diffgraph/processing_modes/__init__.py

```
@@ -0,0 +1,99 @@
+            # Most processors should allow getting description without full initialization
+            temp_instance = processor_class.__new__(processor_class)
+            if hasattr(temp_instance, 'description'):
+                desc = temp_instance.description
+                if isinstance(desc, property):
+                    # For property descriptors, we need to access via class
+                    description = processor_class.description.fget(temp_instance)
+                else:
+                    description = desc
+            else:
+                description = "No description available"
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)